lib/v2/mihoyo/sr/news.js

```diff
@@ -0,0 +1,32 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const locations = {
+    '255': '最新',
+    '256': '新闻',
+    '257': '公告',
+    '258': '活动'
+};
+
+module.exports = async (ctx) => {
+    // location 定位
+    // num 数量：从最新往后
+    const { location = '255', num = '5' } = ctx.params;
```

Comment:
Please use the common parameter `limit` instead.
docs: https://docs.rsshub.app/parameter.html
defined in: https://github.com/DIYgod/RSSHub/blob/1153afc417b994745c29fc2041a43f70624f0cd8/lib/middleware/parameter.js#L157-L159